app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -38,7 +38,8 @@ import org.oppia.android.util.logging.ConsoleLogger
 import org.oppia.android.util.statusbar.StatusBarColor
 import javax.inject.Inject
 
-const val KEY_NAVIGATION_PROFILE_ID = "KEY_NAVIGATION_PROFILE_ID"
+const val NAVIGATION_PROFILE_ID_ARGUMENT_KEY =
+  "NavigationDrawerFragmentPresenter.navigation_profile_id"
```

Comment:
This should be ``NavigationDrawerFragment.navigation_profile_id`` (it's corresponding to the fragment, not the presenter). Ditto elsewhere in this PR.